### DISCUSSION TITLE
Space between the lines

### DISCUSSION THREAD

**Discussion by malu-duarte** [outline/outline#6961] [Ideas]
Hi, guys! 

I have a strange feeling with the space between the lines in Outline.  The space right after a heading isn't enough and when we skip a line, becomes TO MUCH space. This happen between paragrafs too. When we break a paragraph, we expect to give to the reader 'breaths', becoming the text more pleasure to read. The sensation of long paragrafs make the eyes tired and make the reads avoid the text.

E.g on Outline, without break lines, only next line:
![image](https://github.com/outline/outline/assets/113384127/336abf02-8c6e-410e-ab8e-e0ba8ba6267e)

Outline with break lines:
![image](https://github.com/outline/outline/assets/113384127/bf4d37da-2e80-4a5b-89a1-b4c4dffd6665)

And here, the same documentation in Docusaurus, with the break lines visibly smaller:

![image](https://github.com/outline/outline/assets/113384127/5e431182-2fd0-4e85-b47e-49a271040eb8)

Docusaurus Markdown: 
![image](https://github.com/outline/outline/assets/113384127/3de38827-d539-4524-9d23-7a6221cbdb9e)

So, I really don't know how to resolve this, I'm just a technical writer 😅 Maybe leave us configure that? Something like has in the formatation on Google Docs/Word?

![image](https://github.com/outline/outline/assets/113384127/457a6b1e-e4ab-421b-977e-c74b44960b9b)

Anyway, thank you, people! Outline is amazing, and this is only a suggestion to make it even better 💜

**Comment by tommoor** [comment]
Just increased the paragraph spacing from `.5em` -> `.85em`, lets see how that feels. It's hard to make big changes as it does affect the "flow" of peoples existing documents.
